core/src/test/java/org/elasticsearch/index/query/MatchQueryBuilderTests.java

```diff
@@ -196,7 +196,7 @@ protected void doAssertLuceneQuery(MatchQueryBuilder queryBuilder, Query query,
             assertTrue(numericRangeQuery.includesMax());
 
             double value;
-            double width = 0;
+            double width = -1;
             try {
```

Comment:
I think we can also get rid of the try/catch here, from the test setup we know its not date field if `queryBuilder.fieldName().equals(DATE_FIELD_NAME) == false`.